plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar38/python.jjt

```diff
@@ -943,7 +943,20 @@ void flow_stmt() #void: {}
 
 //return_stmt: 'return' [testlist]
 void return_stmt(): {}
-{ begin_return_stmt() [SmartTestList()] }
+{
+
```

Comment:
Minors:

Don't leave that many empty lines around and indent the `[` and `]` to be in its own line (for both edits).

Also, update the template to generate this file.